flink/src/main/java/org/apache/iceberg/flink/IcebergTableSource.java

```diff
@@ -68,13 +64,13 @@ public boolean isBounded() {
 
   @Override
   public TableSource<RowData> projectFields(int[] fields) {
-    return new IcebergTableSource(loader, hadoopConf, schema, options, fields);
+    return new IcebergTableSource(loader, schema, options, fields);
   }
 
   @Override
   public DataStream<RowData> getDataStream(StreamExecutionEnvironment execEnv) {
-    return FlinkSource.forRowData().env(execEnv).tableLoader(loader).hadoopConf(hadoopConf)
-        .project(getProjectedSchema()).properties(options).build();
+    return FlinkSource.forRowData().env(execEnv).tableLoader(loader).project(getProjectedSchema())
+        .properties(options).build();
```

Comment:
nit:   I see somewhere call it `options` and other place call it `properties`, is it possible to unify them ?

Reply:
OK I'll do it.